### DISCUSSION TITLE
Moving The Target Platform of Puppy Linux

### DISCUSSION THREAD

**Discussion by rizalmart** [puppylinux-woof-CE/woof-CE#4228] [Future enhancements / proposals]
Maybe its now time to move the target computer of Puppy Linux. The target hardware of Puppy Linux was late 90's and 2000's PC (aka Windows 9x and XP) era pc. Were now in 2020's those computers are now two decades old compared to the target computer of Puppy was a decade old PC.  It's now time to move the target platform to 2009 model pc (aka Windows 7/8.x/10 era PC) onwards as Windows 11 with high hardware requirements became mainstream and rendered those kind of PC's obsolete despite they have compelling performance.

**Comment by peabee** [comment]
Unsure what you think needs to happen to "move the target computer of Puppy Linux"??
What changes are you suggesting?

**Reply by rizalmart** [reply]
Increase the recommended system requirements for Puppy
1. RAM - from 512 MB to 1GB
2. CPU - from single core to dual core, from 32-bit to 64-bit

**Comment by peabee** [comment]
How? What practical difference would this make?
Are you suggesting changes to:
https://puppylinux-woof-ce.github.io/arch.html
or
https://forum.puppylinux.com/viewtopic.php?t=5127 ? 

> Puppy's come in 32bit and 64bit (newer) versions. This refers to the CPU's ability.
> General hardware minimum system requirements are:
> CPU Type = x86, x86_64, AMD64
> 32bit Puppy's - CPU = single core Pentium 4 or equiv, RAM = 512mb
> 64bit Puppy's - CPU = 64 bit dual core, RAM = 1gb
> 

If so please suggest what.

**Reply by dimkr** [reply]
> https://puppylinux-woof-ce.github.io/arch.html

> Puppy Linux as of November 2015

This page is so outdated. Puppy is i686-only for many years, and a modern Puppy is unusable with 512 MB of RAM.
